pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -215,3 +216,106 @@ func sendCertsProtobuf(attestReq *attest.ZAttestReq, iteration int) {
 			zedcloudCtx, true)
 	}
 }
+
+// conversion routines used while interfacing with controller
+// convertApiHashToLocal :
+func convertApiHashToLocal(hash evecommon.HashAlgorithm) types.ZHashAlgorithm {
```

Comment:
We don't have explicit functions for this. We just type case in zedagent. Lots less code and localized to zedagent.